.maestro/flows/001_002_add_user_direct_test.yaml

```diff
@@ -61,121 +64,177 @@ onFlowComplete:
 - runScript:
     file: "scripts/setDate.js"
 - inputText: ${output.pastYearDateC}
-- hideKeyboard
-
+- runFlow:
+    when:
+      notVisible:
+           id: "phone_countrycode"
+    commands:
+        - hideKeyboard
 # Validate Phone number
 - tapOn:
      id: "phone_countrycode"
 - eraseText
 - inputText: "+1"
 - tapOn: "Phone number"
 - inputText: "23"
-- hideKeyboard
+- runFlow:
```

Comment:
@poojaustad why do we need the hideKeyboard command when save is not visible - the Save button is in the action bar at the top, it will always be visible in the edit screen.

Reply:
<img width="427" height="856" alt="Screenshot from 2025-12-29 18-28-20" src="https://github.com/user-attachments/assets/1d0906f6-220a-467a-825e-1cf600833810" />

@mikedawson When the user enters lower fields like Phone Number or Email, the keyboard covers the screen, making the Save button appear hidden. That’s why the hideKeyboard command is needed to ensure the Save button is visible